test/test_grid_dofhandler_vtk.jl

```diff
@@ -68,8 +69,8 @@ end
         end
         close!(ch)
         update!(ch, 0.0)
-        Random.seed!(1234)
-        u = rand(ndofs(dofhandler))
+        rng = StableRNG(1234)
+        u = rand(rng, ndofs(dofhandler))
```

Comment:
If this is the only place rng is used perhaps just `collect(1:ndofs(dofhandler))`?